libretroshare/src/dht/p3bitdht_relay.cc

```diff
@@ -302,11 +302,11 @@ bool    p3BitDht::loadList(std::list<RsItem *>& load)
 	//config->print(std::cerr, 0);
 
 	std::list<std::string> servers;
-	int peers[RSDHT_RELAY_NUM_CLASS] = {0};
-	int bandwidth[RSDHT_RELAY_NUM_CLASS] = {0};
+	int peers[static_cast<typename std::underlying_type<RsDhtRelayClass>::type>(RsDhtRelayClass::NUM_CLASS)] = {0};
+	int bandwidth[static_cast<typename std::underlying_type<RsDhtRelayClass>::type>(RsDhtRelayClass::NUM_CLASS)] = {0};
 
 	bool haveMode = false;
-	int mode = 0;
+	RsDhtRelayMode mode = static_cast<RsDhtRelayMode>(0);
```

Comment:
same here

Reply:
Fixed! Note: `RsDhtRelayMode` is a bit field / flags type.